src/components/auth/authInput/AuthInput.tsx

```
@@ -0,0 +1,51 @@
+ * @param onChangeText input 현재 value 변경
+ * @returns label과 input을 담은 하나의 요소로 반환
+ */
+export default function AuthInput({ label, placeholder, isError, warningMessage, value, onChangeText }: AuthInputProps) {
+ const [isFocused, setIsFocused] = useState(false);
+
+  return(
+    <TouchableWithoutFeedback onPress={Keyboard.dismiss} accessible={false}>
+    <S.InputBox>
+      {label.length !== 0 && <Typography children={label} font='regular18' color='labelGray' />}
+      <S.Input
+        onFocus={() => setIsFocused(true)}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)